recipes/adios/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "adios" %}
+{% set version = "1.13.1" %}
+{% set sha256 = "684096cd7e5a7f6b8859601d4daeb1dfaa416dfc2d9d529158a62df6c5bcd7a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://users.nccs.gov/%7Epnorbert/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+#   - {{ compiler('fortran') }}
+    - python
+  host:
+    - blosc
+    - bzip2
+    - hdf5 1.10.1
+    - zlib 1.2.11
```

Comment:
I don't think that these need the hard pinnings if your are using the new compilers.  Maybe @conda-forge/staged-recipes can weigh in.